### PR TITLE
Working open3

### PR DESCRIPTION
**Pull request by gcharest**
Added:

- lots of speaker notes
- OSS projects examples
- links to tools 

Changed format of sections.

Related to #322

### CODE REVIEW

**Comment by gcharest** [comment]
Just a note that anyone from the team is welcome to look at this PR. I just put @gabrielcossette and @smellems due to prior work on the topic.

**Comment by remyb2canada** [comment]
For ease of review, use this link (and press "S" to see speaking notes):
https://sara-sabr.github.io/util-presentation/presentation.html?url=https://raw.githubusercontent.com/sara-sabr/ITStrategy/working-open3/presentations/en/2019-11-05-Working-in-the-open.md

**Comment by gcharest** [comment]
@remyb2canada Did you get to read it? :grinning:

**Comment by gcharest** [comment]
Thanks for the review!
> **Additional Comment**
> 
>     * Line 168 : ## How
>       
>       * Content missing or is it the following slides (e.g. Responsibilities)?

I used it as a separator, I'll make a slight pause there for chatting.


>     * Line 222
>       
>       * Doesn't fit in slide

Dang, will have to break it out.
 
>     * Line 277
>       
>       * "expectationsGCcollab"?

Oups, seems like a random copy/paste  

>     * Line 313 : #### Workspace
>       
>       * Add GitHub?

Will double check :thinking:

**Comment by gcharest** [comment]
Fixed the issues!

File: presentations/en/2019-11-05-Working-in-the-open.md [hunk 1]
```diff
@@ -157,30 +256,111 @@ It's still your work, you need to maintain it
...
+#### Workspace
+
 - GCpedia
-- GitLab/GitHub
-- Twitter
-- Open Resources Exchange
-- Slack to a certain extent (closed communities)
+- [GCcollab Wiki](https://wiki.gccollab.ca/)
+- [Framasoft](https://framasoft.org/)
+- [Google Drive + Related Tools](https://www.google.com/drive/)
```
Comment (remyb2canada) [inline presentations/en/2019-11-05-Working-in-the-open.md]: I could see the audience saying "yeah, I'd love that but it's blocked by my department. I am not allowed to use those tools".

To prepare for such questions, I suggest:
1- Policy cover from TBS and raising awareness internally that such dogmatic compliance to other policies may inadvertently affect compliance to broader and more higher up TB policies
2- You are able to use your personnal devices, you do not need to always use your government-provided ones  to do work

File: presentations/en/2019-11-05-Working-in-the-open.md [hunk 2]
```diff
@@ -157,30 +256,111 @@ It's still your work, you need to maintain it
...
+
+#### Communications
+
+- [Twitter](https://twitter.com/)
+- Intranet News and Sites
+- [Open Resources Exchange](https://canada-ca.github.io/ore-ero/)
+
+---
+
+## Examples
```
Comment (remyb2canada) [inline presentations/en/2019-11-05-Working-in-the-open.md]: Suggest adding:
- WET

File: presentations/en/2019-11-05-Working-in-the-open.md [hunk 3]
```diff
@@ -6,56 +6,147 @@
...
+
+Note:
+
+- Many solutions can be useful to problems/people you haven't thought of
+- Contribute to the project so that everyone benefit from it
 
 --
 
->It's all about software engineering economics
+>It's all about software engineericollaborationng economics
```
[the remark this answers is not in the document]
Comment (gcharest) [reply presentations/en/2019-11-05-Working-in-the-open.md]: Dang, I need a new keyboard, it keeps inserting words from the clipboard or double typing letters...

File: presentations/en/2019-11-05-Working-in-the-open.md [hunk 4]
```diff
@@ -157,30 +256,111 @@ It's still your work, you need to maintain it
...
 
+- It should be your source of truth, not a clone of your internal workspace
 - Contributions are welcome but you decide what goes into your work or not
+- It's open source so if it dies in the GC, it may live beyond
 
 --
 
+#### Community
+
 > Warning: A community may coalesce
```
[the remark this answers is not in the document]
Comment (gcharest) [reply presentations/en/2019-11-05-Working-in-the-open.md]: lol!!! I've been hanging with @jaysonmc too much I guess :grin:

File: presentations/en/2019-11-05-Working-in-the-open.md [hunk 5]
```diff
@@ -82,72 +173,80 @@ Note:
...
-- Don't put secrets out there
+- Don't put secrets or sensitive data out there
 - Don't put personnally identifiable information out there
 - Don't put information about procurement processes
-- See list (list to things to not include)
 
 --
 
-#### Public
+#### Public Scrutiny
```
[the remark this answers is not in the document]
Comment (gcharest) [reply presentations/en/2019-11-05-Working-in-the-open.md]: Good point :thinking:

File: presentations/en/2019-11-05-Working-in-the-open.md [hunk 6]
```diff
@@ -157,30 +256,111 @@ It's still your work, you need to maintain it
...
 
-- GCmessage
-  - Instant messages
-- GCcollab
+--
+
+#### Community Management
+
+- [GCmessage (pilot)](https://message.gccollab.ca/home)
+- [Slack](https://slack.com/) to a certain extent (closed communities)
```
[the remark this answers is not in the document]
Comment (gcharest) [reply presentations/en/2019-11-05-Working-in-the-open.md]: GitHub is also proprietary but can be leveraged to foster OSS. Slack is used by OSS project communities but you need to know about it. This presentation is for "working in the open", not just OSS though so... Yeah, YMMV

File: presentations/en/2019-11-05-Working-in-the-open.md (hunk 1 above)
Comment (gcharest) [reply presentations/en/2019-11-05-Working-in-the-open.md]: Good point, I'll add the policies.

As an added context, this is a presentation for the Digital Academy cohort. They have been asked to participate with specific device and permission requirements with the hope to have their home department start to better understand these technologies. We already have seen many for which most of these tools are not available. But the DA will help make the case with their departments, directly with CIOs.

**Comment by gcharest** [comment]
Merci pour les commentaires! Dernière version intégrée!

**Pull request merged**